### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by rayboschbr** [ionic-team/create-stencil#272]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/create-stencil/issues) that already report this problem, without success.

### Create Stencil Version

3.2.0

### Current Behavior

Running `npm init stencil` fails with :

`✖ $t is not a constructor`

### Expected Behavior

The expectation is that it creates a new stencil project starter.

### Steps to Reproduce

Run the following:

` npm install create-stencil@latest`
`npm init stencil`

### Additional Information

OS: Windows 10
Node Version: 18.12.1
NPM Version:  8.19.2

**Comment by rwaskiewicz** [comment]
Hey @rayboschbr 👋 

Thanks for the bug report! I'm unable to reproduce this locally on my Windows machine with those versions of Node and npm. Can you help us out and provide a few additional bits of information for us?

1. What is the output of `npm init stencil -- --info`
2. Can you post the whole stack trace that you're seeing?
3. What starter are you using? Component? App? Something else?
4. Does `npm init stencil@3.2.0 component hello-world` reproduce this issue?

Thanks!

**Comment by rwaskiewicz** [comment]
@rayboschbr By any chance do you have `https_proxy` set an environment variable? I was able to reproduce this error this afternoon by setting that env var like so:
```
https_proxy=1 node index.js component tmp-component-starter 
```

**Comment by rayboschbr** [comment]
@rwaskiewicz yes, I work at a company that uses proxy so need to have both HTTP and HTTPS proxy variables set. 
Is there a way to make it work even behind a corporate proxy wall ?

**Comment by rwaskiewicz** [comment]
@rayboschbr If `https_proxy` is being set, I think that is the issue. This appears to be occurring whenever this global variable is set, we attempt to create a proxy agent based on the value assigned to that environment variable, even if its a valid one.

Can you try using this dev build and let me know if this works for you? 
```
npm init stencil@3.3.0-dev.1690219375.3bb3b0c
```

I'm not sure if you're explicitly setting `https_proxy` or not - if you are, can you use:
```
https_proxy=YOUR_PROXY npm init stencil@3.3.0-dev.1690219375.3bb3b0c
```

**Comment by rayboschbr** [comment]
With this one it works, even tho I'm using the corporate proxy.

**Comment by rwaskiewicz** [comment]
Excellent! I'll put up a pull request with the changes in that dev build for the next release of this CLI

**Comment by rayboschbr** [comment]
Thanks a lot!

**Comment by rwaskiewicz** [comment]
The fix for this issue has been merged and is a [part of the v3.3.1 release](https://github.com/ionic-team/create-stencil/releases/tag/v3.3.1)
